azure-pipelines.yml

```diff
@@ -211,23 +212,39 @@ jobs:
           PYTHON_ARCH: 'x86'
           TEST_MODE: fast
           BITS: 32
+          SCIPY_USE_PYTHRAN: 0
         Python37-64bit-full:
           PYTHON_VERSION: '3.7'
           PYTHON_ARCH: 'x64'
           TEST_MODE: full
           BITS: 64
+          SCIPY_USE_PYTHRAN: 0
         Python38-64bit-full-ilp64:
           PYTHON_VERSION: '3.8'
           PYTHON_ARCH: 'x64'
           TEST_MODE: full
           NPY_USE_BLAS_ILP64: 1
           BITS: 64
           OPENBLAS64_: $(OPENBLAS)
+          SCIPY_USE_PYTHRAN: 0
         Python39-64bit-full:
           PYTHON_VERSION: '3.9'
           PYTHON_ARCH: 'x64'
           TEST_MODE: full
           BITS: 64
+          SCIPY_USE_PYTHRAN: 0
+        Python37-32bit-fast-pythran:
```

Comment:
I'd prefer no extra jobs, we're already struggling with CI load. Let's just set `SCIPY_USE_PYTHRAN` in two existing matrix jobs.